h2o-core/src/main/java/water/fvec/RollupStats.java

```diff
@@ -414,6 +418,8 @@ public void compute2() {
         if (rs != null) {
           if (rs.isReady()) {
             if (_computeHisto && !rs.hasHisto()) { // a.2 => compute rollups
+              final Vec vec = DKV.getGet(_vecKey);
```

Comment:
Ok, I am not that sure about this particular check but I definitely need the second one - I will comment more there.